core/src/main/kotlin/fr/sncf/osrd/api/api_v2/pathfinding/PathfindingBlocksEndpointV2.kt

```diff
@@ -369,3 +385,38 @@ private fun getBlockOffset(
         String.format("getBlockOffset: Track chunk %s not in block %s", trackChunkId, blockId)
     )
 }
+
+/**
+ * Returns the stop position of a train on the signal next to a waypoint block, considering its
+ * rolling stock length.
```

Comment:
As:
* waypoint block is not so obvious name
* it does more than just get the position of the next signal
```suggestion
 * Given a waypoint block (initial position of a train stop), returns the new stop position of a train: next signal, with the constraint of staying in the same block and keeping the tail of the train on the initial position.
 * Note that nodes of the infra are not considered.
```